draft-ietf-tls-wkech.xml

```diff
@@ -825,8 +858,10 @@ Client <----------->|                    | facing
             David Benjamin,
             David Blacka
```

Comment:
`Blacka` ?  Not `Black,` ?

Reply:
Okay, but still needs a trailing comma. :)

Reply:
heh:-)